src/herder/UpgradesTests.cpp

```diff
@@ -282,48 +253,42 @@ testValidateUpgrades(VirtualClock::time_point preferredUpgradeDatetime,
     SECTION("valid fee")
     {
         REQUIRE(Upgrades{cfg}.isValid(checkTime,
-                                      toUpgradeType(makeBaseFeeUpgrade(50)),
-                                      ledgerUpgradeType));
-        REQUIRE(Upgrades{cfg}.isValid(checkTime,
-                                      toUpgradeType(makeBaseFeeUpgrade(200)),
+                                      toUpgradeType(makeBaseFeeUpgrade(100)),
                                       ledgerUpgradeType));
     }
 
     SECTION("too small fee")
     {
         REQUIRE(!Upgrades{cfg}.isValid(checkTime,
-                                       toUpgradeType(makeBaseFeeUpgrade(49)),
+                                       toUpgradeType(makeBaseFeeUpgrade(99)),
```

Comment:
you should collapse those tests: the only thing we're testing is that it's different